lib/service/Service.ts

```diff
@@ -148,9 +148,13 @@ class Service extends EventEmitter {
    * Add an order to the order book.
    */
   public placeOrder = async (order: OwnOrder) => {
-    assert(order.price > 0, 'price must be greater than 0');
+    assert(order.price >= 0, 'price must be greater than 0');
```

Comment:
Change the message here to match the updated logic, maybe `price cannot be negative`.